server/rs_core/templates/registration/login.html

```diff
@@ -1,19 +1,77 @@
 {% extends 'base.html' %}
 
-{% block title %}Login{% endblock %}
+{% block title %}Home{% endblock %}
 
 {% block content %}
-<div class="container">
-  <h2>Login</h2>
-  <form method="post">
-    {% csrf_token %}
-    {{ form.as_p }}
-    <button type="submit">Login</button>
-    <a href="/" class="btn btn-default" type=button">Go back to home page</a>
-  </form>
-
-  <p class="home-page-block">If you have forgotten your password, you can
-      <a href="{% url 'password_reset' %}">reset your password</a>.
-  </p>
-</div>
-{% endblock %}
\ No newline at end of file
+  <div class="container">
+    {% if messages %}
+      <ul>
+        {% for message in messages %}
+        <li {% if message.tags %} class="{{ message.tags }}"{% endif %}>
+          {% if message.level == DEFAULT_MESSAGE_LEVELS.ERROR %}Important: {% endif %}
+          {{ message }}
+        </li>
+        {% endfor %}
+      </ul>
+    {% endif %}
+
+    {% if not user.is_authenticated %}
+      <div class="row">
+        <div class="col-xs-12 col-md-8 offset-md-2">
+          <p>
+            Lorem, ipsum dolor sit amet consectetur adipisicing elit. Ab deserunt, totam. Nulla voluptate,
+            veritatis nostrum laudantium fugiat expedita quo veniam impedit obcaecati atque corporis eius
+            deserunt libero, ex autem, hic corrupti repellat odio aperiam nemo nesciunt. Error reprehenderit
+            nesciunt repudiandae debitis totam fugit ex, voluptates perferendis ratione facilis dolorum, tenetur!
+          </p>
```

Comment:
We'll need to replace these placeholder text with relevant language before merging.

Reply:
agreed. i'll probably just remove it for now then.